packages/webshell/src/features/__tests__/HandleLinkPressFeature.test.tsx

```diff
@@ -37,6 +37,7 @@ describe('Webshell with HandleLinkPressFeature', () => {
       idAttribute: 'anchor0',
       classAttribute: 'link',
       nameAttribute: null,
+      dataAttributes: expect.any(Object),
```

Comment:
Could you test more specifically the exact expected  object? Thanks :-) PR looks good otherwise.

Reply:
Good one, fixed :)